astropy/coordinates/angle_formats.py

```diff
@@ -22,12 +22,13 @@
 
 import numpy as np
 
-from .errors import (IllegalHourWarning, IllegalHourError,
-                     IllegalMinuteWarning, IllegalMinuteError,
-                     IllegalSecondWarning, IllegalSecondError)
+from astropy import units as u
 from astropy.utils import format_exception, parsing
 from astropy.utils.decorators import deprecated
-from astropy import units as u
+
+from .errors import (
+    IllegalHourError, IllegalHourWarning, IllegalMinuteError, IllegalMinuteWarning,
+    IllegalSecondError, IllegalSecondWarning)
```

Comment:
Should we make this `black`-compliant?

Reply:
Fight of the auto tools... But it does bring up a good point that perhaps this should be done as part of what seems likely to be an upcoming black conversion...

Reply:
isort has a black-compliant setting. When the black PR is rolled out, isort will be changed then and will re-sort these commits.
The reason that it's probably better to do isort now and not bundled with black is that import order sometimes matters (see ``coordinates/builtin_frames.__init__.py``).
So my suggestions to just leave this as is and let black do its thing later.

Reply:
Happy with that - nice to separate auto-reformats that could in fact change execution from those that are guaranteed not to.